angr/analyses/decompiler/utils.py

```diff
@@ -712,6 +713,40 @@ def decompile_functions(path, functions=None, structurer=None, catch_errors=Fals
     return decompilation
 
 
+class AILCallCounter(ailment.AILBlockWalkerBase):
```

Comment:
In case you didn't notice, `utils.py` only has methods. Can you move AILCallCounter into a separate file instead?

Reply:
@ltfish yup, I can move it. Do you prefer a single file per class or a type of util-classes file?

Reply:
Let’s put it in its own class file so it follows how other visitor-style classes are implemented under decompiler.

Reply:
@ltfish, lol, imagine forgetting we already had this conversation once...: https://github.com/angr/angr/blob/master/angr/analyses/decompiler/call_counter.py